webtau-graphql-groovy/src/test/groovy/org/testingisdocumenting/webtau/graphql/GraphQLHttpConfigurationsTest.groovy

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2021 webtau maintainers
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.testingisdocumenting.webtau.graphql
+
+import org.junit.After
+import org.junit.Before
+import org.junit.Test
+import org.testingisdocumenting.webtau.graphql.config.GraphQLHttpConfiguration
+import org.testingisdocumenting.webtau.graphql.config.GraphQLHttpConfigurations
+import org.testingisdocumenting.webtau.http.HttpHeader
+import org.testingisdocumenting.webtau.http.config.HttpConfiguration
+import org.testingisdocumenting.webtau.http.config.HttpConfigurations
+
+import static org.testingisdocumenting.webtau.graphql.CustomGraphQLHttpConfiguration.CUSTOM_GRAPHQL_ENDPOINT
+import static org.testingisdocumenting.webtau.graphql.GraphQL.graphql
+
+class GraphQLHttpConfigurationsTest extends GraphQLTestBase {
+    private HttpConfiguration urlVerifier = new HttpConfiguration() {
+        List<String> urls = []
+
+        @Override
+        String fullUrl(String url) {
+            urls << url
+            return url
+        }
+
+        @Override
+        HttpHeader fullHeader(String fullUrl, String passedUrl, HttpHeader given) {
+            return given
+        }
+    }
+    private GraphQLHttpConfiguration withOperationAsQueryParam = new CustomGraphQLHttpConfiguration()
```

Comment:
please add empty line above. One day we will auto check formatting. Maybe in 2025